### ISSUE TITLE
Compositional Splits

### ISSUE THREAD

**Issue by ASMIftekhar** [zhihou7/HOI-CL#21]
Hello,
I hope you are doing well. Where can I find the compositional splits you used in this series of papers? Basically, I want to know which images of HICO-DET go into unseen and which images go into seen splits in the testing set?

**Comment by ASMIftekhar** [comment]
Hello,
I think I have found the splits. However, I just want to make sure. 
1. You basically split the training data, but didn't change the test set. Use same test set all along. Am I correct? 

2. Also, I am not sure about zero_shot_type == 7 here, I think you didn't  report it anywhere in the paper. Correct me if I am wrong. 

https://github.com/zhihou7/FCL/blob/12bc2d4bef4fd2d7d295b33d15f023ae266bc519/lib/ult/ult.py#L398

**Comment by zhihou7** [comment]
Sorry for getting you late, I did not check the email recently. 

1. Yes, I did not change the test set.
2. zero_shot_type ==7 is for the setting when I choose both the tail and head classes for unseen classes. This is for Table 3 in FCL. Sorry for the confusing code.
